src/odemis/gui/cont/tabs.py

```diff
@@ -2595,8 +2595,6 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 raise ValueError("The stage misses the 'SEM_IMAGING_RANGE' metadata.")
             if not model.MD_FM_IMAGING_RANGE in stage_metadata:
                 raise ValueError("The stage misses the 'FM_IMAGING_RANGE' metadata.")
-            if not model.MD_POS_COR in stage_metadata:
-                raise ValueError("The stage misses the 'POS_COR' metadata.")
```

Comment:
Does this not break backwards compatibility? What about the METEOR's in Martinsreid using POS_COR for example?